### ISSUE TITLE
change the elevator scenes to be a bit more interesting

### ISSUE THREAD

**Issue by FamTroder** [FamTroder/FamMel#96]

**Comment by M0nkey5ee** [comment]
Love this scene alltogether! Maybe we could enhance it even more? For example there could be a button on where the whole elevator looks like there are waterfalls on all 4 sides accompanied with aquatic sounds.

**Comment by Forget-About-Me2** [comment]
Potentially, should consider balance between realism though, these are elevators to her apartment. Though could consider adding an elevator to a different location (fancy hotel?)

**Comment by M0nkey5ee** [comment]
True. Although maybe she shares an elevator with a fancy whatever like a museum or similar, so it is still realistic? I think one location should have a defect one where both have to walk upstairs - maybe with the occasional dark corner to pee in or something.
